packages/frontend/src/model/users.ts

```diff
@@ -1,11 +1,25 @@
-import { UserDto, UserRole } from 'api/dist/user/types';
+import { UserDetailsDto, UserDto, UserRole } from 'api/dist/user/types';
 import { AxiosError, AxiosResponse } from 'axios';
-import { atom, selector, selectorFamily, useRecoilState } from 'recoil';
+import {
+  atom,
+  selector,
+  selectorFamily,
+  useRecoilState,
+  useRecoilCallback,
+  useRecoilValue,
+  useSetRecoilState,
+} from 'recoil';
+import React from 'react';
 import { pseudonymNouns, psudonymAdjectives } from '@/utils/users';
 import { useApiAuthClient } from '@/utils/api';
 import { isResponseOk, ApiAuthGet } from './api';
 import { AllPeriods } from './periods';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const instanceOfUser = (object: any): object is UserDto => {
+  return '_id' in object;
```

Comment:
I have used this type of check myself. But, in reality this check does not check if object is a `UserDto` object. Are there any other properties that better identify `UserDto`?